Libraries/Components/Touchable/Touchable.js

```diff
@@ -712,12 +714,21 @@ var TouchableMixin = {
 
       var shouldInvokePress =  !IsLongPressingIn[curState] || pressIsLongButStillCallOnPress;
       if (shouldInvokePress && this.touchableHandlePress) {
+        this._playClickSound();
```

Comment:
No it should just be android. Should I add a Platform.os === 'android' check here?